testkit/src/main/scala/org/bitcoins/keymanager/KeyManagerTestUtil.scala

```diff
@@ -0,0 +1,17 @@
+package org.bitcoins.keymanager
+
+import java.nio.file.Path
+
+import org.bitcoins.core.crypto.AesPassword
+import org.bitcoins.testkit.BitcoinSTestAppConfig
+
+object KeyManagerTestUtil {
+
+  def seedPath: Path = {
+    BitcoinSTestAppConfig
+      .tmpDir()
+      .resolve(WalletStorage.ENCRYPTED_SEED_FILE_NAME)
+  }
+
+  val badPassphrase = AesPassword.fromString("bad-password").get
```

Comment:
I believe I also saw this defined elsewhere, should probably dedup and maybe even just define this val on the `AesPassword` object

Reply:
i made it so it's the same as `KeyManager.badPassphrase`, i do want to keep this val around though as it is useful in test cases.